python/doc/theory/reliability_sensitivity/sensitivity_sobol_from_pce.rst

```diff
@@ -20,24 +20,24 @@ As a result, Sobol' indices can be obtained analytically from the
 coefficients of the PCE (see [sudret2006]_, [sudret2008]_).
 
 Consider the input random vector
-:math:`\vect{X} = \left( X_1,\ldots,X_{n_X} \right)`
-and the output random variable :math:`Y` of the physical model:
+:math:`\inputRV = \left( X_1,\ldots,X_\inputDim \right)`
+and the output random variable :math:`Y` of the model:
 
 .. math::
-    Y = \operatorname{g}(\vect{X}).
+    Y =  \model(\inputRV).
 
 Variance and part of variance of a PCE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let :math:`n_X` be the dimension of the input random vector.
+Let :math:`d` be the dimension of the input random vector.
 Let :math:`P \in \Nset` be the number of coefficients in the functional
 basis.
-Let :math:`\mathcal{J}_P \subseteq \Nset^{n_X}` the set of multi-indices up
+Let :math:`\mathcal{J}_P \subseteq \Nset^{\inputDim}` the set of multi-indices up
 to the index :math:`P`.
 Depending on the way the coefficients are computed, the set of multi-indices
 is the consequence of the choice of the polynomial degree, the enumeration rule,
 and, if necessary, the selection method (e.g. the LARS selection method).
-Let :math:`\tilde{h}(\bdZ)` be the polynomial chaos expansion:
+Let :math:`\metamodel(\standardRV)` be the polynomial chaos expansion:
```

Comment:
```suggestion
Let :math:`\metaModel(\standardRV)` be the polynomial chaos expansion:
```

Reply:
I think thats the error preventing the CI from passing

Reply:
@jschueller : thanks!